specs/resourceGet.yaml

```diff
@@ -0,0 +1,33 @@
+openapi: 3.0.0
+info:
+  title: ResourceGetResponse API
+  version: 1.0.0
+paths:
+  orchestrator/k8s/resource:
+    get:
+      summary: Retrieve resource information
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResourceGetResponse'
+components:
+  schemas:
```

Comment:
there should be an existing spec for this, if present please update there and remove this

Reply:
DONE